tada_a_bigoh.py

```diff
@@ -100,15 +132,15 @@
                     ratio = mean / meanlastround
                     avg = (mean + meanlastround) / 2
                     std = abs(mean - avg)
-                    indicator = std / avg
+                    indicator = std / abs(avg)
```

Comment:
Is the absolute average still necessary here since we wouldn't get any negative average value if we are not doing subtraction time?

Reply:
I think although with the right approach, we won't get negative value for either std and avg, but having both of them in absolute value can prevent some possible problems. It also makes more sense if we have both of them in absolute value instead of only std to calculate indicator.